### PR TITLE
Changes kops vpc plugin to amazonvpc for presubmit scale job

### PR DESCRIPTION
**Pull request by ashishranjan738**
This commit changes kops vpc plugin to amazonvpc for presubmit scale jobs.

### CODE REVIEW

**Comment by shyamjvs** [comment]
/lgtm
/hold

cc @BenTheElder or @justinsb - for approval

File: config/jobs/kubernetes/kops/build_jobs.py [hunk 1]
```diff
@@ -1036,6 +1037,7 @@ def generate_presubmits_scale():
         presubmit_test(
             name='presubmit-kops-aws-scale',
```
Comment (hakman) [inline config/jobs/kubernetes/kops/build_jobs.py:1038]:
```suggestion
            name='presubmit-kops-aws-scale-amazonvpc',
```

File: config/jobs/kubernetes/kops/build_jobs.py [hunk 2]
```diff
@@ -1023,6 +1023,7 @@ def generate_scale():
             name_override='kops-aws-scale',
```
Comment (hakman) [inline config/jobs/kubernetes/kops/build_jobs.py:1023]:
```suggestion
            name_override='kops-aws-scale-amazonvpc',
```

**Review by hakman — CHANGES_REQUESTED** [review]
Let's add CNI as suffix. We will want to add some tests with other CNIs until things work as expected.
/cc @justinsb

File: config/jobs/kubernetes/kops/build_jobs.py (hunk 2 above)
Comment (justinsb) [reply config/jobs/kubernetes/kops/build_jobs.py:1023]: Should we have both jobs for a bit?

**Comment by justinsb** [comment]
I'm also OK though if we want to just tweak this job, get it working, then create a few jobs to cover a few scenarios!

**Comment by ashishranjan738** [comment]
@justinsb @hakman I have updated the job names. Let me know if we are good for merge

**Comment by justinsb** [comment]
Thanks @ashishranjan738 

/approve
/lgtm
/hold cancel

**Pull request merged**